.github/workflows/terraform_security.yml

```diff
@@ -0,0 +1,18 @@
+name: "Terraform security validation workflow"
+
+on: workflow_call
+
+jobs:
+  validate:
+    name: Terraform validate
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: terraform
+    env:
+      AWS_REGION: "eu-west-1"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: tfsec
+        uses: aquasecurity/tfsec-action@v1.0.0
```

Comment:
Can you pass through the GITHUB_TOKEN as well and bump the version to 1.0.3? It occasionally rate limits otherwise.

See https://github.com/communitiesuk/delta-common-infrastructure/blob/main/.github/workflows/check.yml#L35, we should update it for Delta too at some point.